app/src/main/java/com/goryachok/forecastapp/view/recyclerviewadapters/HourlyForecastAdapter.kt

```diff
@@ -25,17 +24,17 @@ class HourlyForecastAdapter(private var forecast: List<Forecast>) :
         )
     }
 
-    override fun getItemCount(): Int = forecast.size
+    override fun getItemCount(): Int = forecast.size / 5
```

Comment:
Please remove magic number, you can move to constants or remove variable.

Reply:
Removed